### PR TITLE
Added Manual Installation guide

### LINKED ISSUE #25: Manual Installation
`CocoaPods` and `Carthage` are awesome tools and make our life really easier, but there are some devs who still don't know how to use them. 

It would be cool to add the _Manual installation_ guide in your `README.md`. You can take a look at my [iOS Readme Template](https://github.com/awesome-labs/iOS-readme-template) to see how you can do it.

### PR DESCRIPTION
**Pull request by lfarah**
fixes #25

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,10 @@ github "crossroadlabs/Regex"
 
 Run `carthage update` and follow the steps as described in Carthage's [README](https://github.com/Carthage/Carthage#adding-frameworks-to-an-application).
 
+#### Manually
+1. Download and drop ```/Regex``` folder in your project.
```
Comment (dileping) [inline README.md:56]: Could you, please, specify where from to download it?

**Comment by dileping** [comment]
@lfarah Thanks a lot for your contribution! Hope my comment makes sense. What do you think?

File: README.md (hunk 1 above)
Comment (lfarah) [inline README.md:56]: You would download the `Zip` from Github:
screen shot 2016-06-03 at 11 43 18 am

**Comment by dileping** [comment]
@lfarah Could you, please, accept the CLA on CLAhub so I could merge it? Thanks

**Comment by lfarah** [comment]
I think I signed already

**Comment by dileping** [comment]
@lfarah Just double checked and I don't see it signed for this project. Could you double check or redo on your end, please? Thanks!

**Comment by lfarah** [comment]
screen shot 2016-06-08 at 1 58 58 pm

**Comment by dileping** [comment]
Whatever. Merging it in

**Comment by lfarah** [comment]
Thanks!

**Pull request merged**